locuszoom.app.js

```diff
@@ -267,6 +267,11 @@ LocusZoom.createCORSPromise = function (method, url, body, timeout) {
         };
         timeout && setTimeout(response.reject, timeout);
         body = typeof body !== "undefined" ? body : "";
+        // If posting an object set the proper content-type header
+        if (method == "POST"){
+            xhr.setRequestHeader("Content-Type","application/x-www-form-urlencoded");
```

Comment:
Yeah, I think there is just a `response`, not sure why I used `resposeText` originally.